### PR TITLE
Feature: Saga Actions, Flux Standard Actions; Action handling helpers

### PR DESCRIPTION
**Pull request by MartynasZilinskas**
We should normalize functions/properties naming in a few places (please check also tests naming). @DovydasNavickas

### CODE REVIEW

File: packages/saga/src/__tests__/dispatcher.test.ts [hunk 1]
```diff
@@ -1,101 +1,108 @@
...
+    dispatchToken3 = dispatcher.register(store3);
 
-    dispatcher.dispatch(TEST_ACTION);
+    dispatcher.dispatch(new TestAction());
 
-    expect(callOrder).toEqual([STORE_NAME3, STORE_NAME2, STORE_NAME1]);
+    expect(callOrder).toEqual([dispatchToken3, dispatchToken2, dispatchToken1]);
 });
 
 it("waitFor must throw while not being in middle of dispatch", () => {
```
Comment (DovydasNavickas) [inline packages/saga/src/__tests__/dispatcher.test.ts]:
```suggestion
it("waitFor must throw while not being in the middle of dispatch", () => {
```

File: packages/saga/src/actions.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ * Returns `true` if `action` is FSA compliant.
+ */
+export function isFSA<TPayload, TMeta = undefined>(action: any): action is FSA<TPayload, TMeta> {
+    return typeof action === "object" && typeof action.type === "string" && Object.keys(action).every(isValidKey);
+}
+
+/**
+ * Returns `true` if `action` is FSA compliant error.
+ */
+export function isErrorFSA<TCustomError extends Error, TMeta = undefined>(action: any): action is ErrorFSA<TCustomError, TMeta> {
```
Comment (DovydasNavickas) [inline packages/saga/src/actions.ts]: `isFSAError`

File: packages/saga/src/__tests__/store.test.ts [hunk 3]
```diff
@@ -1,56 +1,85 @@
...
-                }
-                case "COUNTER_DECREMENT": {
-                    return {
-                        counter: state.counter - 1
-                    };
-                }
-                default: {
-                    return state;
-                }
+        // reducer: (state, action) => {
```
[the remark this answers is not in the document]
Comment (MartynasZilinskas) [reply packages/saga/src/__tests__/store.test.ts]: I wanted to leave that comment as an example. I think it should belong in examples (docs) then in tests.

File: packages/saga/src/actions.ts (hunk 2 above)
Comment (MartynasZilinskas) [reply packages/saga/src/actions.ts]: Changed to `isErrorAction`.

**Pull request merged**